.github/workflows/release-artifacts.yml

```diff
@@ -47,8 +47,7 @@ jobs:
         id: build
         shell: bash
         run: |
-          choco install openssl
-          export OPENSSL_DIR="C:\Program Files\OpenSSL-Win64"
+          export OPENSSL_DIR="C:\Program Files\OpenSSL"
```

Comment:
confirming this is correct change as these are essentially same as similarly affected github ci users: https://github.com/open-quantum-safe/liboqs-rust/pull/198/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR44